packages/create-next-app/helpers/examples.ts

```diff
@@ -82,50 +88,50 @@ export function existsInRepo(nameOrUrl: string): Promise<boolean> {
   }
 }
 
-async function downloadTar(url: string) {
-  const tempFile = join(tmpdir(), `next.js-cna-example.temp-${Date.now()}`)
-  await pipeline(got.stream(url), createWriteStream(tempFile))
-  return tempFile
+async function downloadTarStream(url: string) {
+  const resp = await fetch(url)
+
+  if (!resp.body) {
+    throw new Error(`Failed to download: ${url}`)
+  }
+
+  return Readable.fromWeb(resp.body as import('stream/web').ReadableStream)
```

Comment:
```suggestion
  const res = await fetch(url)

  if (!res.body) {
    throw new Error(`Failed to download: ${url}`)
  }

  return Readable.fromWeb(res.body as import('stream/web').ReadableStream)
```

Reply:
Addressed in [`786162b` (#58030)](https://github.com/vercel/next.js/pull/58030/commits/786162b26394a37fcf1a2d63c3cedc99a5efb911).